introduction.md

```diff
@@ -1,2 +1,9 @@
 JavaScript is my favorite code school path so far.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?